src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +84,17 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    @Override
+    public String getPrintableString(Printable... printables) {
+        String returnMe = "";
+        for(Printable p : printables) {
+            returnMe += p.getPrintableString();
```

Comment:
This works but using `StringBuilder` is much better then `+=` cause it allows `.append()` on String. Google it to know more about it